testsuite/tests/tool-lexyacc/calc_parser.mly

```diff
@@ -9,7 +9,11 @@
 %type <int> main
 %%
 main:
-    expr EOL                { $1 }
+    expr EOL                {
+            (* {A|  *)
+            (* {a| *) } } }  |a} *)
+            (* {%%A.ba| *) } } }  |} *)
+            $1 }
```

Comment:
I would rather a separate .mly file to test advanced parsing questions, and keep the basic test simple and readable.